config/config.go

```diff
@@ -169,6 +173,20 @@ type Support struct {
 	Email string `yaml:"email,omitempty" json:"email,omitempty" validate:"omitempty,email"`
 }
 
+// +kubebuilder:object:generate=true
+type MetadataLink struct {
+	// Name of the metadata collection/site/organization
+	Name string `yaml:"name" json:"name" validate:"required"`
+
+	// Which part of the API this metadata concerns. E.g. dataset (in general), tiles or features
```

Comment:
Zou er tiles, features, geovolumes van maken om aan te geven dat het alles mag zijn.

Deze "part" staat neem ik aan los van de parts in de ogc specs, want dat is want anders.